backend/src/main/java/ca/bc/gov/hlth/hnweb/model/eligibility/InquirePhnRequest.java

```diff
@@ -1,9 +1,10 @@
-package ca.bc.gov.hlth.hnweb.model;
+package ca.bc.gov.hlth.hnweb.model.eligibility;
 
+import java.util.ArrayList;
 import java.util.List;
 
 public class InquirePhnRequest {
-	private List<String> phns;
+	private List<String> phns = new ArrayList<String>();
```

Comment:
Do we want to start conforming, as just discussed, to use just the <> here?

Reply:
Yes. I blame Eclipse.